gtd-timer/Services/LogInService.cs

```diff
@@ -0,0 +1,63 @@
+﻿using Microsoft.IdentityModel.Tokens;
+using System;
+using System.Security.Claims;
+using System.Text;
+using gtdtimer.Timer.DAL.Entities;
+using gtdtimer.Timer.DTO;
+using System.Threading.Tasks;
+using gtdtimer.Constant;
+using System.IdentityModel.Tokens.Jwt;
+
+namespace gtdtimer.Services
+{
+    public class LogInService : ILogInService
+    {
+        private ApplicationUserManager<User> userManager;
```

Comment:
service should use UoW to access user manager

Reply:
Already changed